src/main/java/org/synyx/urlaubsverwaltung/person/basedata/PersonBasedataRepository.java

```diff
@@ -0,0 +1,6 @@
+package org.synyx.urlaubsverwaltung.person.basedata;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+interface PersonBasedataRepository extends JpaRepository<PersonBasedataEntity, Integer> {
```

Comment:
why jpaRepository? Do we use jpa internas? Or should a CRUD be enough?

Reply:
Replaced